torchci/clickhouse_queries/master_commit_red/query.sql

```diff
@@ -0,0 +1,70 @@
+--- This query is used to show the histogram of trunk red commits on HUD metrics page
+--- during a period of time
+WITH all_jobs AS (
+  SELECT
+    push.head_commit.'timestamp' AS time,
+    CASE
+      WHEN job.conclusion = 'failure' THEN 'red'
+      WHEN job.conclusion = 'timed_out' THEN 'red'
+      WHEN job.conclusion = 'cancelled' THEN 'red'
+      WHEN job.conclusion IS NULL THEN 'pending'
```

Comment:
job.conclusion is not nullable

Reply:
Ok, do you know what would happen if the job is still pending here? It has `pending` as a value?

Reply:
previously null values should get the default for their type, in this case just empty string